CHANGELOG.md

```diff
@@ -3,14 +3,10 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
 ## Unreleased
 
-### Added
-- 
-
```

Comment:
This should have a header for 3.0.9 and the placeholders for Unreleased should stay in place.